### PR TITLE
doc(release): Release notes for web 24.04.2 / dsm 24.04.2

### PR DESCRIPTION
**Pull request by pkippes**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [x] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Review by rsadik-ext — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx [hunk 1]
```diff
@@ -22,6 +22,29 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [API] Fixed access control for some v2 API endpoints.
+- [UX] Removed autocomplete for credentials fields in administration pages.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [CEIP] Fixed an error on Dashboard telemetry.
+- [Unattended] unattended.sh script now allows new options (Debian12, MariaDB versionalization, MySQL 8)
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx:41]:
```suggestion
- [Unattended] The unattended.sh script now allows new options (Debian12, MariaDB versioning, MySQL 8).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx [hunk 2]
```diff
@@ -22,6 +22,29 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Unattended] bug fixes on various unattended.sh issues (httpd services not restarting, it-edition-extensions installation, script run issue on Oracle Linux distributions).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx:42]:
```suggestion
- [Unattended] Bug fixes on various unattended.sh issues (httpd services not restarting, it-edition-extensions installation, script run issue on Oracle Linux distributions).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx [hunk 3]
```diff
@@ -22,6 +22,29 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Unattended] the installation and the activation of Centreon IT Edition Extensions is now available
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx:43]:
```suggestion
- [Unattended] The installation and the activation of Centreon IT Edition Extensions is now available
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx [hunk 4]
```diff
@@ -22,6 +22,29 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
 
+### 24.04.2
+
+Release date: `May 29, 2024`
+
+<details open>
+  <summary>New features</summary>
+
+- [API] Fixed access control for some v2 API endpoints.
+- [UX] Removed autocomplete for credentials fields in administration pages.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx:33]: these are not new features

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**